tex/chapters/gausss-law/infinite-plane.tex

```diff
@@ -0,0 +1,31 @@
+\section{Infinite plane}
+An infinite plane with charge density per unit of area $\sigma$ in empty space, located on the $XY$-plane is given. Due to the symmetry of a
+system, the electric field is constant and in the direction of the $z$-axis.
+
+
+\be\label{infinite-plane-e-field}
+\mathbf{E} = E_{z}(z)\uvec{z} = E_{z}(\sign(z))\uvec{z}
+\ee
+
+By replacing \ref{infinite-plane-e-field} in \ref{gauss-law}, we can write
+\be\label{infinite-plane-flux}
+\oiint_A E_{z}(\sign(z))\uvec{z} \cdot \mathrm{d}\mathbf{A} =\frac{Q_{encl}}{\epsilon_0}
+\ee
+where $\mathrm{d}\mathbf{A} = \mathrm{d}x \mathrm{d}y$ is infinitesimal area with the direction normal to the surface and parallel to the
+$z$-axis.
+
+Considering a parallelepiped centered at the location of the charged plane, the total flux equals to the sum of the fluxes of each face. 
+
+\be
+\Phi_{E_{tot}}=\Phi_{x_{+}}+\Phi_{x_{+}}+\Phi_{y_{+}}+\Phi_{y_{+}}+\Phi_{z_{+}}+\Phi_{z_{-}}
+\ee
+Flux is calculted as a dot product between $\mathbf{E}$ and $\total{\mathbf{A}}$, but because electric field is in the $z$-direction, the contributions to the flux in $x$- and $y$-direction are zero. So we have
+\be\label{}
+\oiint_A E_z\frac{z}{|z|}\uvec{z} \cdot\uvec{z} \total{x}\total{y}+
```

Comment:
What is $z$? You probably mean the z-coordinate of the "top" and "bottom" planes.

Reply:
done